sparse_strips/vello_hybrid/examples/winit/src/render_context.rs

```diff
@@ -183,19 +183,17 @@ impl RenderContext {
     async fn new_device(&mut self, compatible_surface: Option<&Surface<'_>>) -> Option<usize> {
         let adapter =
             wgpu::util::initialize_adapter_from_env_or_default(&self.instance, compatible_surface)
-                .await?;
+                .await
+                .expect("No suitable GPU adapters found on the system!");
```

Comment:
I'd expect this to use `.ok()?` like the later one.

Reply:
Fixed.